sfs/mono/source.py

```diff
@@ -79,6 +79,54 @@ def point_velocity(omega, x0, n0, grid, c=None):
     return util.XyzComponents([v * o / r for o in offset])
 
 
+def point_dipol(omega, x0, n0, grid, c=None):
+    """Point source with dipole characteristics.
+
+    Parameters
+    ----------
+    omega : float
+        Frequency of source.
+    x0 : (3,) array_like
+        Position of source.
+    n0 : (3,) array_like
+        Normal vector (direction) of dipole.
+    grid : triple of numpy.ndarray
+        The grid that is used for the sound field calculations.
+        See :func:`sfs.util.xyz_grid`.
+    c : float, optional
+        Speed of sound.
+
+    Returns
+    -------
+    numpy.ndarray
+        Sound pressure at positions given by `grid`.
+    ::
```

Comment:
The docstring is for the description of the behavior of the function, that's exactly where the equation should be.
Comments are for implementation-specific remarks.
Since Python is such a readable language, comments can be quite rare ...

You cannot move the equation to the last section, because that's `Examples`, where it doesn't belong.